### ISSUE TITLE
MPI_T: Errata: Reinitialization

### ISSUE THREAD

**Issue by kathrynmohror** [mpi-forum/mpi-issues#50]
# Problem

In 3.1, page 571 line 14: When discussing how MPI_T can be reinitialized with another call to MPI_T_Init_thread, we say:
The interface can be reinitialized by subsequent calls to MPI_T_INIT_THREAD.

This is not technically correct, as only the first call to MPI_T_Init_thread will reinitialize the tools interface.

# Proposal

Replace 3.1, page 571 line 14: 
The interface can be reinitialized by subsequent calls to MPI_T_INIT_THREAD.

with 

The interface can be reinitialized by the next call to MPI_T_INIT_THREAD.

# Changes to the Text

See above.

# Impact on Implementations

None

# Impact on Users

Clarification

# References

**Comment by dholmes-epcc-ed-ac-uk** [comment]
I would also suggest changing "can be" to "will be", unless the tools interface is allowed to refuse to re-initialise when instructed to do so.

**Comment by kathrynmohror** [comment]
We do allow initialization to fail, and an implementation could return MPI_T_ERR_CANNOT_INIT to indicate this.

**Comment by kathrynmohror** [comment]
[Pull request 20](https://github.com/mpi-forum/mpi-standard/pull/20)

**Comment by wgropp** [comment]
Done.  Thanks
Bill

William Gropp
Director, Parallel Computing Institute
Thomas M. Siebel Chair in Computer Science
Chief Scientist, NCSA
University of Illinois Urbana-Champaign

**Comment by mahermanns** [comment]
Created new pull request mpi-forum/mpi-standard/pull/25

**Comment by kathrynmohror** [comment]
Due to confusion in the voting record from September 2016, we re-read this ticket at the March 2017 meeting. At this meeting, the forum asked us to change the text to be clearer. The new replacement text is: 
The user can reinitialize the interface by a subsequent call to MPI_T_INIT_THREAD.

We will read the ticket again at the next meeting in June.

**Comment by kathrynmohror** [comment]
Created new pull request: [Pull request 28](https://github.com/mpi-forum/mpi-standard/pull/28)

**Comment by wesbland** [comment]
I know I'm late to the party here, but you don't have to open a new pull request every time you want to update the commit. You can just force push to your branch in the mpiwg-tools fork. I believe both you and Marc-Andre have push access there.

**Comment by mahermanns** [comment]
Yes, we are aware of that, but the problem was that we kind of messed up that branch, and we could not squash the commits into a single commit anymore. It seemed easier for Kathryn and me to just create a new PR with the small change that was left after all the different commits. (see [XKCD #1597](https://xkcd.com/1597/))

**Comment by wesbland** [comment]
Should I give you my phone number for your git.txt? 😆

**Comment by schulzm** [comment]
Passed errata vote on 9/22
Can be added to the errata document right away
